03-User-Sessions/Pods/Auth0/README.md

```diff
@@ -214,6 +214,26 @@ credentialsManager.credentials { error, credentials in
 }
 ```
 
+#### Clearing credentials and revoking refresh tokens
```

Comment:
Seeing this duplicated, I'm wondering ... is this really necessary in the README? Could it be comments in the code instead and keep the README focused on running the sample?